src/Controller/CommentsController.php

```diff
@@ -254,16 +255,25 @@ private function url(string $itemType, Comment $comment): ?string
      *
      * @return JsonResponse
      */
-    private function addComment(Request $request, string $itemType, $item, UserInterface $user): Response
+    private function addComment(Request $request, string $itemType, $item, UserInterface $user, Aws $aws): Response
```

Comment:
`$aws` can probably be added to the constructor and not need to be passed here (or into `newDocCommentAction`.